src/ui/pages/SidePage/components/WalletConnect/WalletConnectStageTwo.tsx

```diff
@@ -87,7 +91,6 @@ const WalletConnectStageTwo = ({
           );
         }
 
-        dispatch(setIsConnecting(true));
```

Comment:
Actually, this is used for a different reason. It gets set back to false once the pending connection is set. I wouldn't mess with this. Can we just add a separate `useState` for `[startingMeerkat, setStartingMeerkat]` and use this instead? It's less risky...